### PR TITLE
feat: add instructions to install vault operator and vault instance chart in k3s

### PR DESCRIPTION
**Pull request by ankitm123**
Signed-off-by: ankitm123 <ankitmohapatra123@gmail.com>

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# Checklist:

- [ ] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
- [ ] I have signed off the commit, as per instructions mentioned [here](https://jenkins-x.io/community/code/#how-to-sign-your-commits).
- [ ] Any dependent changes have already been merged.

### CODE REVIEW

File: content/en/v3/admin/platforms/k3s/_index.md [hunk 1]
```diff
@@ -30,8 +29,7 @@ Make sure you have created a cluster using k3s.
 If you dont have an existing k3s cluster, you can install one by running:
 
 ```bash
-# We don't support Kubernetes 1.22+ yet
-curl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL=v1.21 sh -s - --write-kubeconfig-mode 644
+curl -sfL https://get.k3s.io | sh -s - --write-kubeconfig-mode 644
```
Comment (ankitm123) [inline content/en/v3/admin/platforms/k3s/_index.md:32]: We support the latest k3s version which runs kubernetes 1.23

File: content/en/v3/admin/platforms/k3s/_index.md [hunk 2]
```diff
@@ -55,29 +53,44 @@ export KUBECONFIG=~/.kube/config:~/.kube/k3s-config
...
+Check [k3s install guide](https://rancher.com/docs/k3s/latest/en/installation/) for more installation options.
 
 #### Vault
 
 Install vault cli.
 Refer to the [vault docs](https://www.vaultproject.io/docs/install) on how to install vault for your platform.
 
+##### Internal vault (Preferred)
+
+To install vault inside the newly created k3s cluster, you need to install the vault operator and vault instance chart.
```
Comment (babadofar) [inline content/en/v3/admin/platforms/k3s/_index.md:79]: why not add these in a helmfile folder named jx-vault  and install using `helmfile sync jx-vault`

**Comment by babadofar** [comment]
Perhaps we could consolidate this with this ancient PR before it decomposes https://github.com/jenkins-x/jx-docs/pull/3588

File: content/en/v3/admin/platforms/k3s/_index.md (hunk 2 above)
Comment (ankitm123) [reply content/en/v3/admin/platforms/k3s/_index.md:79]: I am fine if sync works, I have not tried it yet.
The issue is that vault needs to be installed before gitops operator is running.

**Pull request merged**